### ISSUE TITLE
Chevron on primary menu items may not meet contrast criteria in Windows high contrast/forced colors mode

### ISSUE THREAD

**Issue by trimspa** [uswds/uswds#4453]
### Describe the bug

The chevron on individual items in the mega menu may not meet color contrast criteria when using Window's High Contrast Mode. 

Windows high contrast lets you select multiple themes (black on white, white on black) or create custom themes. The chevron's background color does not respond to the theme changing colors. If you have a white on black theme (the default setting) the color of the icon is barely distinguishable from the background color. 

### Steps to reproduce the bug

1. Turn on Windows High Contrast Mode.
2. Ensure you are using a white on black theme.
2. Go to the [Header](https://designsystem.digital.gov/components/header/) component on the USWDS website. 
3. Click on the [Basic Header](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/preview/header--basic.html). 
4. Click on the component preview.
5. You will be unable to see the chevron (indicating a drop down menu) unless you hover your mouse over it. 

### Expected behavior
Given that this chevron conveys visual meaning that aids user navigation, it should be visible not matter the chosen theme. 

### Screenshots

<img width="563" alt="Screen Shot 2022-01-07 at 3 38 06 PM" src="https://user-images.githubusercontent.com/64163603/148613527-203f7e87-8ca7-4aef-98aa-27cdf29a6a5f.png">

### System setup
- **USWDS version:**
- **Device: MacBook Pro**
- **Operating system: Windows 10 (Parallels), Big Sur 11.6**
- **Browser and version: Edge**

### Additional context

We/I will be filing a PR for this issue next week. Just wanted to add the issue here for tracking purposes.

**Comment by mherchel** [comment]
Hey USWDS team 👋 

Our team at AVB/Bixal is going to be working on this issue shortly as part of the SBA.gov work. In order to proceed we need a bit of guidance on which technical fix we should implement.

Potential fixes:
1. We can inline the SVG into the HTML, and then force the background color using something like `forced-colors: active` and `background-color: LinkText`. This would have backwards compatibility issues because we force users to update their HTML to include the SVG.
2. We can inline the SVG via JavaScript, and use a technique similar to the one above
3. We can include a `:after` pseudo-element and use borders to make the chevron. Borders show fine in forced colors mode, and is a normal solution.
4. We can try something like [CSS clip-path](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path) along with changing the background color via media query and keywords (similar to the first option).

My preference would be 3. It's a normal pattern to use pseudo-elements to make shapes and it works really well cross-browser (including IE11). 

Thoughts?

**Comment by mahoneycm** [comment]
Related: #4467 

Just wanted to share that I've found the same issue appears on all components which utilize a drop-down such as time picker and combo box.

I'm toying around with some solutions as well. I was hoping to do something similar to [Icon](https://designsystem.digital.gov/components/icon/) where the SVG changes colors on hover, but make it a static color when `forced-colors: active` is set. I was hoping to find a place where we can set it once and have it work for all dropdowns but I'm having trouble getting it to work so your solutions may be the better route!

**Comment by mherchel** [comment]
Yeah, things like this are a pain. 

If the SVG is inline to the HTML, changing the color via CSS works fine, but as soon as it's a background image (like in the menu chevrons), you can no longer change the color. It's basically treated the same as a PNG or JPG. :(

**Comment by mejiaj** [comment]
Thank you @trimspa and @mherchel for reporting this and the contribution w/ dropdowns!

There's a newer mixin that uses the masking technique, with a fallback that uses masks to set the color. You can find that in [`_icon.scss`](https://github.com/uswds/uswds/blob/716cd4de52159bbba022c93431043ca39e7702a8/src/stylesheets/core/mixins/_icon.scss#L73) it's called `add-color-icon`. You can see how that's used on external-links [here](https://github.com/uswds/uswds/blob/716cd4de52159bbba022c93431043ca39e7702a8/src/stylesheets/core/mixins/_external-link.scss#L17)

Alternatively, seems like a fine solution. Especially if you use `currentColor` to define the color. But I'd try the mask option first.
>We can include a :after pseudo-element and use borders to make the chevron. Borders show fine in forced colors mode, and is a normal solution.

**Comment by mherchel** [comment]
Thanks for the comment! 

Note that there's a difference between [CSS Mask](https://developer.mozilla.org/en-US/docs/Web/CSS/mask) and [CSS clip-path](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path).

That being said, the CSS mask technique and mixin looks like it might work and also utilize the same SVG (which is nice). 

We'll give that a try.

**Comment by mahoneycm** [comment]
Resolved by #4604
